Sources/Tracing/Docs.docc/Guides/InstrumentYourLibrary.md

```diff
@@ -298,9 +298,9 @@ It is worth noting that double-ending a span should be considered a programmer e
 >
 > Please also take care to never `end()` a span that was created using `withSpan()`  APIs, because `withSpan` will automatically end the span when the closure returns.
 
-#### Storing and restoring baggage across callbacks 
+#### Storing and restoring context across callbacks 
 
-Note also since a `Span` contains an instrumentation `Baggage`, you can also pass the span's baggage to any APIs which may need it, or even restore the baggage e.g. for loggers to pick it up while emitting log statements:
+Note also since a `Span` contains an instrumentation `ServiceContext`, you can also pass the span's context to any APIs which may need it, or even restore the context e.g. for loggers to pick it up while emitting log statements:
```

Comment:
```suggestion
Note also since a `Span` contains a `ServiceContext`, you can also pass the span's context to any APIs which may need it, or even restore the context e.g. for loggers to pick it up while emitting log statements:
```